app/src/main/java/com/framgia/fsalon/screen/booking/BookingFragment.java

```diff
@@ -35,13 +40,20 @@ public static BookingFragment newInstance() {
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        mViewModel = new BookingViewModel(getActivity());
+        Activity activity = getActivity();
+        if(activity instanceof AdminBookingActivity){
+            BookingOder order = activity.getIntent().getExtras().getParcelable(Constant
+                .BUNDLE_ORDER);
+            mViewModel = new BookingViewModel(getActivity(), order);
+        } else {
+            mViewModel = new BookingViewModel(getActivity());
+        }
         BookingContract.Presenter presenter =
             new BookingPresenter(mViewModel,
                 new BookingRepository(
                     new BookingRemoteDataSource(FSalonServiceClient.getInstance())),
-                new SalonRepository(
-                    new SalonRemoteDataSource(FSalonServiceClient.getInstance())),
+                new SalonRepository(new SalonRemoteDataSource
+                    (FSalonServiceClient.getInstance())),
```

Comment:
ERROR: 'operator new lparen' have incorrect indentation level 20, expected level should be 16.